lib/bibliothecary/parsers/rubygems.rb

```diff
@@ -30,10 +30,11 @@ def self.mapping
 
       def self.parse_gemfile_lock(manifest)
         manifest.split("\n").map do |line|
```

Comment:
I think we can rely on ruby's https://apidock.com/ruby/String/each_line. Maybe give that a shot and run a couple scenarios through it to make sure it's true?

Reply:
Seems that doesn't actually strip the chars though, still need to remove them
```
irb(main):002:0> "alksdjf\nlaksdjf\r\nlaksjdf\nlkajsdf\rlkjasdlfkj".each_line { |l| puts l.dump }
"alksdjf\n"
"laksdjf\r\n"
"laksjdf\n"
"lkajsdf\rlkjasdlfkj"
=> "alksdjf\nlaksdjf\r\nlaksjdf\nlkajsdf\rlkjasdlfkj"
```